src/main/resources/assets/cofhworld/lang/en_US.lang

```diff
@@ -0,0 +1,6 @@
+
+reload.usage=Reloads CoFHWorld configuration
+reload.successful=Configuration reloaded.
+reload.failed=Failed to reload config; check your logs.
+list.usage=Lists all CoFHWorld features
+list=Available CoFHWorld features: %s
```

Comment:
Localizations are global, so make sure all of our stuff is prefixed like core does. That way we avoid collisions with other mods.